docs/feature_converters.md

```diff
@@ -93,6 +96,30 @@ The following defaults are based on what has been implemented for [RP2040](platf
 | USB Host (e.g. USB-USB converter)            | Not supported (USB host code is AVR specific and is not currently supported on ARM)                              |
 | [Split keyboards](feature_split_keyboard.md) | Partial via `PIO` vendor driver - heavily dependent on enabled features                                          |
 
-### SparkFun Pro Micro - RP2040 and Blok :id=promicro_rp2040 
+### SparkFun Pro Micro - RP2040 and Blok :id=promicro_rp2040
 
 Currently identical to  [Adafruit KB2040](#kb2040).
+
+### STeMCell :id=stemcell
+
+Feature set currently identical to [Proton C](#proton_c).
+
+There are two versions of STeMCell available, with different pinouts:
+  - v1.0.0 - STMC_VERSION=1
+  - v1.0.1 - STMC_VERSION=2 (default)
+
+The default version selected is v1.0.1. To compile for v1.0.0, use the following flag while compiling.
+
+```make
+-e STMC_VERSION=1
```

Comment:
Not a fan of these compile time flags, feels a lot like the previous CTPC short hand which was removed. (Though im not sure right now on how best to handle the case)

I also have some concerns on versioning being implemented as defines, but thats going to take some time to increment within the current framework.

Reply:
Do you have any suggestion on how to improve this ? 
Or shall I remove the old version to support only current version for now ?

Reply:
Looks like the pin swap was to better accommodate SPI pinout, correct? Normally when an external interface changes in an incompatible way (like for a pin swap) you would increment the major version number (e.g. 1.0.0 -> 2.0.0) to signify that backwards compatibility is broken.

Since that isn't what happened here, I would lean towards supporting only the current version in the QMK official repo, yet also providing a separate branch in your QMK fork to support anyone using the older version of the board. As it currently stands, anyone using the old version board has already had to make do without the converter being in the QMK repo, so basically, nothing much would change for them: they can just keep on using a branch in your fork that is dedicated to the older version.

Reply:
Yep, make sense, Also I guess most people have 1.0.1/1.0.2 version.
So i will remove the code supporting older version 1.0.0

Reply:
We'll have to think up a solution w.r.t. the compile-time flags... but for now it shouldn't hold this PR up.